floris/tools/optimization/layout_optimization/layout_optimization_boundary_grid.py

```diff
@@ -8,6 +8,12 @@
     Polygon,
 )
 
+from floris.tools import (
+    TimeSeries,
+    WindRose,
+)
+from floris.tools.wind_data import WindDataBase
+
```

Comment:
It doesn't seem like these WindData classes are used in this file. Are they needed?

Reply:
You're right, deleting